src/utils/bootstrapUtils.js

```diff
@@ -126,6 +130,10 @@ export function getClassSet(props) {
   return classes;
 }
 
+export function omitBsProps(props) {
```

Comment:
No – each component should own deleting the props it knows about. This is just a convenience method for the shared stuff.